examples/proc_combined_ODNP.py

```diff
@@ -0,0 +1,332 @@
+from Instruments.logobj import logobj
+logger = init_logging("info")
+Ep_signal_pathway = {'ph1':1}
+fl=fl_mod()
+filename='211213_10mM_TEMPOL_test'
+file_location = 'ODNP_NMR_comp/ODNP'
+Ep_f_slice=(0.15e3,0.4e3)
+IR_f_slice=(0.15e3,0.4e3)
+Ep_t_range = (0,0.2)
+t_range = (0,0.2)
+excluded_pathways = [(0,0)]
+R1w = 1/2.172
+nPowers = 15
+ppt = 1.5167e-3
+C=0.01
+#{{{load in log
+with h5py.File(search_filename(filename+".h5",exp_type='ODNP_NMR_comp/ODNP',unique=True),'r') as f:
+    log_grp = f['log']
+    thislog = logobj()
+    thislog.__setstate__(log_grp)
+    read_array = thislog.total_log
+    read_dict = thislog.log_dict
+for j in range(len(read_array)):
+    thistime,thisrx,thispower,thiscmd = read_array[j]
+fig, (ax_Rx,ax_power) = plt.subplots(2,1, figsize=(10,8))
+fl.next("log figure",fig=fig)
+ax_Rx.set_ylabel('Rx/mV')
+log_start_time = read_array['time'][0]
+print("LOG START TIME IS:",log_start_time)
+relative_time = read_array['time']
+ax_Rx.plot(relative_time, read_array['Rx'],'.')
+ax_power.set_ylabel('power/dBm')
+ax_power.plot(relative_time,read_array['power'],'.')
+mask = read_array['cmd'] != 0
+n_events = len(relative_time[mask])
+trans_power = transforms.blended_transform_factory(
+        ax_power.transData,ax_power.transAxes)
+trans_Rx = transforms.blended_transform_factory(
+        ax_Rx.transData, ax_Rx.transAxes)
+for j, thisevent in enumerate(read_array[mask]):
+    ax_Rx.axvline(x=thisevent['time']-log_start_time)
+    ax_power.axvline(x=thisevent['time']-log_start_time)
+    y_pos = j/n_events
+    #ax_Rx.text(thisevent['time']-log_start_time,y_pos,read_dict[thisevent['cmd']],transform=trans_Rx)
+    #ax_power.text(thisevent['time']-log_start_time,y_pos,read_dict[thisevent['cmd']],transform=trans_power)
+ax_power.legend(**dict(bbox_to_anchor=(1.05,1),loc=2,borderaxespad=0.))
+plt.tight_layout()
+power_axis = nddata(read_array['power'],[-1],['time'])
+power_axis.setaxis('time',relative_time)
+power_axis.setaxis('time',lambda x: x - log_start_time)
+fl.next('power axis')
+fl.plot(power_axis,'.')
+power_axis = nddata(read_array['power'],[-1],['time'])
+power_axis.setaxis('time',relative_time)
+power_axis.name('power')
+fl.next('power axis W')
+coupler_atten = 22
+power_axis.data = (10**((power_axis.data+coupler_atten)/10+3))/1e6 #convert to W
+fl.plot(power_axis,'.')
+#}}}
+#{{{IR processing
+T1_list = []
+power_list = []
+start_times = []
+stop_times = []
+errors=[]
+for nodename, postproc, signal_pathway, clock_correction in [
+        ('FIR_noPower','spincore_IR_v1',{'ph1':0,'ph2':1},False),
+        ('FIR_27dBm','spincore_IR_v1',{'ph1':0,'ph2':1},False),
+        ('FIR_30dBm','spincore_IR_v1',{'ph1':0,'ph2':1},False),
+        ('FIR_32dBm','spincore_IR_v1',{'ph1':0,'ph2':1},False),
+        ('FIR_33dBm','spincore_IR_v1',{'ph1':0,'ph2':1},False),
+        ]:
+    IR = find_file(filename,exp_type=file_location,expno=nodename,
+            postproc=postproc,lookup=lookup_table)
```

Comment:
big problem here -- postproc argument should not be used or required -- your ppg should be setting the appropriate attribute of the nddata so that it knows which postproc to use.  and the name should NOT be version 1, since you are using the log, etc, etc, which is not part of version 1!

Reply:
version 1 is literally just reordering and fting the time and phase cycling dimensions in order to display the raw data. Why is that not okay for this processing?

Reply:
but I edited the ppg to store the postproc for IR as an acq_params called IR_postproc...will this new method that I have load it correctly? I don't have data where it has this parameter since I JUST added it

Reply:
what is the name of the property that you are setting now, and what's it set to? (so that I can check against pyspecdata)

Reply:
see [here](https://github.com/jmfrancklab/spincore_apps/pull/8/files#r771069613)